### ISSUE TITLE
508 Issues from pa11y-rails

### ISSUE THREAD

**Issue by nickbristow** [18F/myra#183]
Results can be recreated using [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/)
## Results for https://myra.treasury.gov
- **Error** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1.88:1. Recommendation: change text colour to #005d85.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - `#footer-site > section:nth-child(1) > ul > li:nth-child(2) > a > span`
  - `<span class="sr-only">Social Security</span>`
- **Error** Img element inside a link must not use alt text that duplicates the text content of the link.
  - WCAG2AA.Principle1.Guideline1_1.1_1_1.H2.EG5
  - `#footer-site > section:nth-child(1) > ul > li:nth-child(1) > a > img`
  - `<img src="/images/icons/irslogo_white.png" alt="IRS">`
- **Error** Img element inside a link must not use alt text that duplicates the text content of the link.
  - WCAG2AA.Principle1.Guideline1_1.1_1_1.H2.EG5
  - `#footer-site > section:nth-child(1) > ul > li:nth-child(2) > a > img`
  - `<img src="/images/icons/ssalogo.jpg" alt="Social Security">`
- **Error** Duplicate id attribute value "" found on the web page.
  - WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
  - `html > body > script:nth-child(9)`
  - `<script type="text/javascript" id="">(function(){var b=document.getE...</script>`
- **Error** Duplicate id attribute value "" found on the web page.
  - WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
  - `html > body > script:nth-child(11)`
  - `<script type="text/javascript" id="">!function(b,e,f,g,a,c,d){b.fbq|...</script>`
- **Error** Duplicate id attribute value "" found on the web page.
  - WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
  - `html > body > script:nth-child(13)`
  - `<script type="text/javascript" id="">(function(){var a=window,b=docu...</script>`
- **Error** Img element missing an alt attribute. Use the alt attribute to specify a short text alternative.
  - WCAG2AA.Principle1.Guideline1_1.1_1_1.H37
  - `html > body > img`
  - `<img src="https://secure.leadback.advertising.com/adcedge/lb?site=695501&amp;srvc=1&amp;betr=53037=1403859[8760]" width="1" height="1" border="0">`
- **Error** Iframe element requires a non-empty title attribute that identifies the frame.
  - WCAG2AA.Principle2.Guideline2_4.2_4_1.H64.1
  - `html > body > iframe`
  - `<iframe src="https://5387330.fls.doubleclick.net/activityi;src=5387330;type=myrah0;cat=myrah0;dc_lat=;dc_rdid=;tag_for_child_directed_treatment=;ord=[SessionID]?" width="1" height="1" frameborder="0" style="display:none"></iframe>`
- **Error** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.03:1. Recommendation: change text colour to #007eb6.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - `#calculator > div > div:nth-child(4) > a`
  - `<a href="/savings-calculators#disclaimer" onmousedown="_sendEvent('Downloads','savi','https://www.myra.gov/savings-calculators',0);">See calculation details</a>`
- **Error** Iframe element requires a non-empty title attribute that identifies the frame.
  - WCAG2AA.Principle2.Guideline2_4.2_4_1.H64.1
  - `#rfi_tc_ifr_20722526`
  - `<iframe id="rfi_tc_ifr_20722526" border="0" frameborder="0" vspace="0" hspace="0" scrolling="no" marginheight="0" marginwidth="0" style="display:none;padding:0;margin:0" width="0" height="0" src="https://20722526p.rfihub.com/ca.html?ver=9&amp;rb=2455...`
- **Error** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1.88:1. Recommendation: change text colour to #005d85.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - `#footer-site > section:nth-child(1) > ul > li:nth-child(1) > a > span`
  - `<span class="sr-only">IRS</span>`
### Summary

**Errors:** 11
